app.py

```diff
@@ -210,18 +210,13 @@ def _schedule_core_solver_jobs(openshift: OpenShift, index_urls: List[str], resu
 def cli(verbose: bool = False, dry_run: bool = False, result_api: str = None, index_base_url:
         str = None, register_indexes_only: bool = False):
     """Register AICoE indexes in Thoth's database."""
-    graph = None
```

Comment:
This will cause issues when referencing (unexisting) `graph` variable in:

```python
    indexes = _register_indexes(graph, index_base_url, dry_run)
```

Reply:
rechkpls